.obs/chartfile/operator/questions.yaml

```diff
@@ -1,24 +1,24 @@
 questions:
-- variable: defaultChannels.sleMicro55.included
-  show_if: "defaultChannels.sleMicro55"
+- variable: defaultChannels.slMicro60Baremetal.included
```

Comment:
This `slMicro60Baremetal` is matching the values.yaml keys. The same applies for the other channels.